definitions/divorce/json/CaseEvent.json

```diff
@@ -2049,5 +2060,39 @@
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
     "SecurityClassification": "Public",
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "DATimeTriggerPassed",
+    "Name": "DA Time Trigger",
```

Comment:
I'd make the event name not related to "time trigger".
Even though it is triggered when an amount of time passes, this responsibility lies in the job scheduler, not CCD.
As far as CCD is concerned, all this event does is make the case eligible for the petitioner to apply for DA.

Reply:
Sure, happy to change